src/app/resources/iis_scripts/token_utils.ps1

```diff
@@ -62,16 +62,7 @@ function TokenRequest([string] $targetEndpoint, [string]$method, [string]$subpat
 if ($tokenId) {
     $existingToken = @{ "id" = $tokenId }
 } else {
-    try {
-        $query = Invoke-WebRequest "$apiHost/security/$CreateEndpoint" -UseBasicParsing -UseDefaultCredentials -ContentType "application/json"
-    } catch {
-        if ($_.Exception.Status -eq [System.Net.WebExceptionStatus]::ConnectFailure) {
-            ## NOTE: we will be detecting this error message upstream
-            throw "Unable to connect to the remote server"
-        } else {
-            throw $_
-        }
-    }
+    $query = Invoke-WebRequest "$apiHost/security/$CreateEndpoint" -UseBasicParsing -UseDefaultCredentials -ContentType "application/json"

```

Comment:
Since we are chaning the flow to invoke admin api util first, we would detect the non-existance in admin api util instead